File: Heat/ViewModels/Plants/AddContactPlantViewModel.vb
Imports System.ComponentModel.DataAnnotations
Imports Heat.Models

Namespace ViewModels.Plants
    Public Class AddContactPlantViewModel
        Property PlantID As Integer

        <Required> _
        <Display(name:="Nominativo")> _
        Property Name As String

        <Display(name:="Tipo contatto")> _
        Property AddressTypeID As Integer
        Property AddressTypeList As IEnumerable(Of SelectListItem)

        <Display(name:="Indirizzo")> _
        Property Street As String

        <Display(name:="Numero civico")> _
        Property StreetNumber As String

        <Display(name:="Località")> _
        Property City As String

        <Display(name:="CAP")> _
        Property PostalCode As String

        <Display(name:="Provincia")> _
        Property District As String


        <Display(name:="Note")> _
        <DataType(DataType.MultilineText)> _
        Property Note As String

        <Display(name:="Telefono")> _
        Property Phone As String

        <Display(name:="Cellulare")> _
        Property CellPhone As String

        <Display(name:="Fax")> _
        Property Fax As String

        <Display(name:="Email")> _
        <DataType(DataType.EmailAddress)> _
        Property Email As String

        <Display(name:="Web")> _
        <DataType(DataType.Url)> _
        Property URL As String


    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/Plants/AddContactPlantViewModel.cs
using System.Collections.Generic;
using System.Web.Mvc;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Plants
{
    public class AddContactPlantViewModel
	{
		public int PlantID { get; set; }

		[Required()]
		[Display(Name = "Nominativo")]
		public string Name { get; set; }

		[Display(Name = "Tipo contatto")]
		public int AddressTypeID { get; set; }
		public IEnumerable<SelectListItem> AddressTypeList { get; set; }

		[Display(Name = "Indirizzo")]
		public string Street { get; set; }

		[Display(Name = "Numero civico")]
		public string StreetNumber { get; set; }

		[Display(Name = "Località")]
		public string City { get; set; }

		[Display(Name = "CAP")]
		public string PostalCode { get; set; }

		[Display(Name = "Provincia")]
		public string District { get; set; }


		[Display(Name = "Note")]
		[DataType(DataType.MultilineText)]
		public string Note { get; set; }

		[Display(Name = "Telefono")]
		public string Phone { get; set; }

		[Display(Name = "Cellulare")]
		public string CellPhone { get; set; }

		[Display(Name = "Fax")]
		public string Fax { get; set; }

		[Display(Name = "Email")]
		[DataType(DataType.EmailAddress)]
		public string Email { get; set; }

		[Display(Name = "Web")]
		[DataType(DataType.Url)]
		public string URL { get; set; }


	}

}
